src/smart-components/request/request-table-helpers.js

```diff
@@ -10,27 +10,31 @@ export const createRows = (data) =>
     name,
     requester_name,
     created_at,
-    updated_at,
+    notified_at,
+    finished_at,
     state,
     decision,
     reason,
     number_of_children,
-    content }, key) => ([
+    content
+  }, key) => ([
     ...acc, {
       id,
       isOpen: false,
       state,
       number_of_children,
       cells: [ <Fragment key={ id }><Link to={ `/requests/detail/${id}` }>
         <Button variant="link"> { name } </Button></Link></Fragment>, requester_name,
-      timeAgo(created_at), timeAgo(updated_at), state, decision ]
+      timeAgo(created_at), finished_at ? timeAgo(finished_at) : timeAgo(notified_at), state, decision ]
```

Comment:
You should use the DateTime from insights-frontend-components. We want to remove `moment` if possible.